sources/install/package_ad.sh

```diff
@@ -1141,7 +1143,7 @@ function install_netexec() {
     git -C /opt/tools/ clone --depth 1 https://github.com/Pennyw0rth/NetExec
     pipx install /opt/tools/NetExec/
     mkdir -p ~/.nxc
-    [ -f ~/.nxc/nxc.conf ] && mv ~/.nxc/nxc.conf ~/.nxc/nxc.conf.bak
+    [[ -f "~/.nxc/nxc.conf" ]] && mv ~/.nxc/nxc.conf ~/.nxc/nxc.conf.bak
```

Comment:
Tilde doesn't expand in quotes. Quotes need to be removed.

Reply:
Done